src/components/auth/SignInScreen.js

```diff
@@ -103,104 +103,138 @@ export default class SignInScreen extends React.Component {
                       onChangeText={value => this.onChangeText('password', value)}
                     />
                   </Item>
+                  {/* Sign Up Text */}
+                  <Text style={styles.buttonText2}>Novo usuário?{" "}
+                    <Text 
+                      onPress={() => this.props.navigation.navigate('SignUp')}
+                      style={styles.buttonText2link}>Crie uma conta 
+                    </Text>
+                    {" "}»
+                  </Text>
+                  {/* Forgot Password Text */}
+                  <Text style={styles.buttonText2}>Esqueceu sua senha?{" "}
+                    <Text
+                      onPress={() => this.props.navigation.navigate('ForgotPassword')}
+                      style={styles.buttonText2link}>Recupere aqui
+                    </Text>
+                    {" "}»
+                  </Text>
+                  {/* Loading ActivityIndicator */}
+                </View>  
+                <KeyboardAvoidingView style={styles.container2} behavior='position' enabled>
                   {/* Sign In Button */}
                   <TouchableOpacity
                     onPress={() => this.signIn()}
                     disabled={this.state.isLoading}
                     style={styles.buttonStyle1}>
+                  {!this.state.isLoading &&
                     <Text style={styles.buttonText1}>
-                      {i18n.t('Sign In')}
-                    </Text>
-                  </TouchableOpacity>
-                  {/* Sign Up Text */}
-                  <TouchableOpacity
-                    onPress={() => this.props.navigation.navigate('SignUp')}
-                    style={styles.buttonStyle2}>
-                    <Text style={styles.buttonText2}>
-                      {i18n.t('New User? Create an Account!')}
-                    </Text>
-                  </TouchableOpacity>
-                  {/* Forgot Password Text */}
-                  <TouchableOpacity
-                    onPress={() => this.props.navigation.navigate('ForgotPassword')}
-                    style={styles.buttonStyle2}>
-                    <Text style={styles.buttonText2}>
-                      {i18n.t('Forgot Password?')}
+                      Entrar
                     </Text>
-                  </TouchableOpacity>
-                  {/* Loading ActivityIndicator */}
+                  }
                   {this.state.isLoading &&
                     <View>
-                      <ActivityIndicator color={COLORS.lightblue} size='large' animating={this.state.isLoading} />
+                      <ActivityIndicator color={COLORS.sandy} size='large' animating={this.state.isLoading} />
```

Comment:
I did forest green on sign in, should I swap or should you swap?

Reply:
Use his. His activity indicator is set into his Button (replaces the text) which i think looks really good. Already left a comment on yours to match this